policyengine/countries/us/reform_parameters.yaml

```diff
@@ -140,6 +140,26 @@ reforms:
         variable: snap_dependent_care_deduction
         tests:
           - increases_net_income: false
+    abolish_ma_dependent_care_credit:
+      values:
+        2010-01-01: false
+      metadata:
+        label: Abolish MA dependent care credit
+        name: abolish_ma_dependent_care_credit
+        unit: abolition
+        variable: ma_dependent_care_credit_amount
+        tests:
+          - increases_net_income: false
+    abolish_wa_wftc:
+      values:
+        2010-01-01: false
+      metadata:
+        label: Abolish WA Working Families Tax Credit
+        name: abolish_wa_wftc
+        unit: abolition
+        variable: wa_wftc_max_amount
```

Comment:
Does this work, given [`wa_wftc_max_amount` is a scale parameter](https://github.com/PolicyEngine/openfisca-us/blob/master/openfisca_us/parameters/gov/states/wa/tax/income/credits/working_families_tax_credit/amount.yaml)?

Reply:
I should probably point it to a variable instead of a parameter then!